src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -1276,6 +1302,25 @@ public void process(HttpMessage msg) {
 		}
 	}
 
+	private static List<Integer> getIds(JSONObject params) throws ApiException {
+		List<Integer> listIds = new ArrayList<>();
+		for (String id : params.getString(PARAM_IDS).split(",")) {
+			String trimmedId = id.trim();
```

Comment:
Any reason not to do the split and trim at the same time, which would avoid the extra var?